app/code/Magento/Catalog/view/frontend/templates/product/list/items.phtml

```diff
@@ -147,7 +147,11 @@ switch ($type = $block->getType()) {
     break;
 
     case 'other':
+        $exist = null;
     break;
+        
+    default:
+        $exist = null;
```

Comment:
Rather than adding the default options here what about adding an `isset` check as part of the if wrapping the content display.

Reply:
Changed as requested. Your suggestion is a neater solution.